src/TEdit.Terraria/World.Properties.cs

```diff
@@ -795,15 +795,17 @@ public byte Bg8
     [Reactive] public byte BgTree4 { get; set; }
     [Reactive] public byte UnderworldBg { get; set; }
     [Reactive] public byte MushroomBg { get; set; }
-
-
-
+	
     [Reactive] public int TilesWide { get; set; }
+	[Reactive] public int TilesHighReactive { get; set; }

```

Comment:
The seems redundant, but I could be missing something.

Reply:
Due to how `[Reactive]` works, and being it's get and set control members are declared in another module, this was the only feasible workaround to this I personally could find without it getting crazy.